### ISSUE TITLE
Setting the urgency flag via wmctrl doesn't update the tag_status

### ISSUE THREAD

**Issue by jsteel44** [herbstluftwm/herbstluftwm#586]
Running something like:

$ wmctrl -r Chromium -b add,demands_attention

Causes Chromium to have the urgency flag, so using:

$ herbstclient jumpto urgent

switches focus to the application, however (before focusing the urgent window):

$ herbstclient tag_status

Doesn't show that the workspace Chromium is on has the urgent flag. If Chromium is visible (but not focused) then I see the window border change colour based on the urgency flag, so it seems to just be an issue with the tag status. Is there any way to manually trigger an update of the tag status at least?

Thanks!

**Comment by t-wissmann** [comment]
Thanks for the bug report, I can reproduce it.

You can manually trigger the tag status update with the command:
```
herbstclient chain , add dummyTagName , merge_tag dummyTagName
```

**Comment by jsteel44** [comment]
Thanks for that, I was going to add it to my script that sets the urgency flag, however herbstluftwm also fails to notice removal of the urgency flag (when I focus the window), so I'll probably just run this command in a loop every 5 seconds or so until this is resolved. Thanks for your help.

**Comment by jsteel44** [comment]
Interestingly enough, it works fine with some applications but not others. A QT5 one works without needing to update the tag status.

**Comment by t-wissmann** [comment]
Sorry for not reacting for so long. I've fixed it in the git version (#892 / 98f8df6), so this will be fixed in the next release.
